### PR TITLE
Completed term entry for JS .padStart()

### PR DESCRIPTION
**Pull request by nbarnhouse**
### Description

Completed term entry for JavaScript String term .padStart(). 

### Issue Solved

Closes #issue-number 4727

### Type of Change

- Adding a new entry

### Checklist

- [ x] All writings are my own.
- [ x] My entry follows the Codecademy Docs style guide.
- [ x] My changes generate no new warnings.
- [ x] I have performed a self-review of my own writing and code.
- [ x] I have checked my entry and corrected any misspellings.
- [ x] I have made corresponding changes to the documentation if needed.
- [ x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [ x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [ x] I have linked any issues that are relevant to this PR in the `Issues Solved` section.

### CODE REVIEW

File: content/javascript/concepts/strings/terms/padStart/padStart.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+Subjects:
+  - 'Web Development'
+  - 'Computer Science'
+Tags:
+  - 'Strings'
+  - 'Methods'
+CatalogContent:
+  - 'introduction-to-javascript'
+  - 'paths/front-end-engineer-career-path'
+---
```
Comment (mamtawardhani) [inline content/javascript/concepts/strings/terms/padStart/padStart.md:13]:
```suggestion
---
Title: '.padStart()'
Description: 'Returns a new string of a specified length with the original string padded from the start with a given string.'
Subjects:
  - 'Computer Science'
  - 'Web Development'
Tags:
  - 'Methods'
  - 'Strings'
CatalogContent:
  - 'introduction-to-javascript'
  - 'paths/front-end-engineer-career-path'
---
```

File: content/javascript/concepts/strings/terms/padStart/padStart.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+
+Takes an original string and returns a second string to the front in order to change the original string to a specified length.
```
Comment (mamtawardhani) [inline content/javascript/concepts/strings/terms/padStart/padStart.md:15]:
```suggestion
In JavaScript, the **`.padStart()`** method returns a new string of a specified length, with the original string padded at the beginning with a given string (repeated if necessary) until the desired length is reached.
```

File: content/javascript/concepts/strings/terms/padStart/padStart.md [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+
+## Syntax
+
+```pseudo
+originalString.padStart(targetLength, padString);
```
Comment (mamtawardhani) [inline content/javascript/concepts/strings/terms/padStart/padStart.md:20]:
```suggestion
string.padStart(targetLength, padString);
```

File: content/javascript/concepts/strings/terms/padStart/padStart.md [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+```
+
+- `originalString` starting string value before padding.
+- `targetLength` indicates the length the resulting string should be.
+- `padString`, (optional) character or string value to pad the original string.
```
Comment (mamtawardhani) [inline content/javascript/concepts/strings/terms/padStart/padStart.md:25]:
```suggestion
- `string`: The original string to be padded.
- `targetLength`: The desired length of the resulting string after padding.
- `padString`: The string to pad the original string with. If not provided, it defaults to a space (' ').
```

File: content/javascript/concepts/strings/terms/padStart/padStart.md [hunk 5]
```diff
@@ -0,0 +1,98 @@
...
+```js
+const string = 'help!';
+const targetLength = 9
+const padString = 'Can you '
+
+const paddedString = string.padStart(targetLength, padString);
+
+console.log(paddedString);
+// Output: Can help!
+```
```
Comment (mamtawardhani) [inline content/javascript/concepts/strings/terms/padStart/padStart.md:76]: I would suggest putting all these use cases in one example, you can have multiple `console.log` showcasing multiple outputs.
After this bloc, use the ```shell``` block to show the output, avoid writing the output comments in the code itself. You can have comments in the code to explain the particular padding.

File: content/javascript/concepts/strings/terms/padStart/padStart.md [hunk 6]
```diff
@@ -0,0 +1,98 @@
...
+
+## Codebyte Example
+
+The following is runnable, and demonstrates the use of the `.padStart()` method:
```
Comment (mamtawardhani) [inline content/javascript/concepts/strings/terms/padStart/padStart.md:80]:
```suggestion
The following example demonstrates the use of the `.padStart()` method:
```

**Review by mamtawardhani — CHANGES_REQUESTED** [review]
Hey @nbarnhouse! Thank you for your contribution to Codecademy Docs! The entry was nicely written!

I've suggested a few changes for the entry and added some comments for the ##Example section, could you please review and modify those at your earliest convenience? Feel free to discuss any concerns.
Thank you! 😄

**Review by nbarnhouse — COMMENTED** [review]
please let me know if there is anything else I need to adjust @mamtawardhani.

**Review by mamtawardhani — APPROVED** [review]
Thank you for making the required changes @nbarnhouse!

This looks good for a second review! 😃

**Review by letsbuildanything — APPROVED** [review]
Ready to be Merged! 🚀

**Pull request merged**